velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +92,29 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseLaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double p) {
+    if (std::isnan(location) || std::isnan(scale) || std::isnan(p)) {
+      result = std::numeric_limits<double>::quiet_NaN();
+    } else if (scale <= 0 || p < 0 || p > 1) {
+      result = std::numeric_limits<double>::quiet_NaN();
+      VELOX_USER_CHECK_GT(scale, 0, "scale must be greater than 0");
```

Comment:
Its strange that these CHECKS happen after the assignment of the result. I think they are meant for the cases in the else part of the condition below.

Please can you check.